pike/test/copychunk.py

```diff
@@ -504,7 +511,7 @@ def generic_ssc_overlap_test_case(self, block, number_of_chunks, overlap_each_bl
         self._create_and_write(src_filename, block)
 
         total_len = len(block)
-        chunk_sz = (total_len / number_of_chunks) + 1
+        chunk_sz = (old_div(total_len, number_of_chunks)) + 1
```

Comment:
What's the diff between `old_div` and `//`?

Reply:
no difference AFAICT, futurize uses `old_div` to be explicit. i think perhaps not every python programmer has encountered `//`, so old_div at least gives _some_ idea of whats going on. i'll probably leave this for now

Reply:
switched `old_div` to `//` throughout, it looks way nicer